### ISSUE TITLE
Documentation:  Enhancement to ADTTE vignette for complex parameters

### ISSUE THREAD

**Issue by manciniedoardo** [pharmaverse/admiralonco#288]
### Please select a category the issue is focused on?

Documentation

### Let us know where something needs a refresh or put your idea here!

We've had feedback from Roche users that the ADTTE vignette does not cover enough examples for complex censoring rules, in particular that `derive_param_tte()` is not sufficient for some of their derivations. If working at Roche, see [here](https://overflow.roche.com/t/adtte-with-different-censoring-rules/18873), where eventually some discussion led to the proposal to use `derive_extreme_event()` in conjunction with various `event()` objects.

@bundfussr Could the ADTTE vignette could be enhanced to point users in this direction if they have similar use cases? Maybe make a new "Advanced" section mentioning that this is a possibility, with an example to showcase?

I'm not an expert in the TA so leave to you to figure out the details of what and how to showcase.

**Comment by bundfussr** [comment]
I think this is not an oncology-specific issue.

In most cases time-to-event analyses are done for negative events like death, progression, AE... The `derive_param_tte()` function and the vignettes were designed for such cases. However, the team which started the discussion wanted to derive a TTE endpoint for a positive event (time to first response). In this case the censoring rules are slightly different. I would suggest to update the TTE vignette in admiral and enhance `derive_param_tte()` if necessary.

@manciniedoardo , what do you think?

**Comment by manciniedoardo** [comment]
@bundfussr thanks for your response. Given your expertise and suggestions, how about the following:

- Update TTE vignette in admiral package as described above (ie. referencing possible uses of `derive_extreme_event()` for positive events)
- Link to that in TTE vignette in admiralonco (just a small section saying something along the lines of "If you find yourself needing to derive an endpoint for a positive event, eg. time to first response, check out the following example here)
- No updates to `derive_param_tte()`

**Comment by bundfussr** [comment]
I agree to update the TTE vignettes. However, I'm not sure if we should use `derive_extreme_event()` or enhance `derive_param_tte()`. I think this should depend on what is easier to use and read.

There is also another use case which is not covered by the vignettes. Setting `EVNTDESC` for censoring is sometimes tricky. For example if you are deriving time to `CHG >= 10` and want to distinguish subjects censored because they don't have a baseline value and subject censored because they don't have post-baseline values. I think we should add something for this use case as well.

I would consider both use cases first and try out how they could be solved with existing functions or how they could be solved with enhanced functions. And then we could decide which way we want to go.

@manciniedoardo , does that make sense?

**Comment by bundfussr** [comment]
This should be covered by the updates in admiral 1.5.0 (update of `derive_param_tte()` and new "Time-to-event Analyses" vignette). The admiralonco TTE vignette was updated accordingly in https://github.com/pharmaverse/admiralonco/issues/348.
